cwms_radar_api/src/main/java/cwms/radar/api/TimeSeriesController.java

```diff
@@ -356,7 +356,10 @@ public void getOne(Context ctx, String id) {
     }
 
     @OpenApi(
-            description = "Update a TimeSeries",
+            description = "Update a TimeSeries with provided values",
+            pathParams = {
+                @OpenApiParam(name="name",description = "Full CWMS Timeseries name")
```

Comment:
We should review the swagger docs and decide on TimeSeries, Timeseries, Time series, Time-series.  I'm definitely guilty of being inconsistent.